detox/local-cli/test.js

```diff
@@ -240,16 +240,22 @@ module.exports.handler = async function test(program) {
   }
 
   function runJest() {
-    if (platform === 'android' && program.workers !== 1) {
-      log.warn('Can not use -w, --workers. Parallel test execution is only supported on iOS currently');
-      program.w = program.workers = 1;
+    const hasMultipleWorkers = ((program.workers || 0) > 1);
```

Comment:
Why `|| 0`? Was there something wrong with `type: number, default: 1` handling in yargs?